serverkeyapi/internal/api.go

```diff
@@ -33,10 +65,12 @@ func (s *ServerKeyAPI) StoreKeys(
 	// Run in a background context - we don't want to stop this work just
 	// because the caller gives up waiting.
 	ctx := context.Background()
+
 	// Store any keys that we were given in our database.
 	return s.OurKeyRing.KeyDatabase.StoreKeys(ctx, results)
 }
 
+// nolint:gocyclo
```

Comment:
Uhh, this is a 150 line function. Large functions make it harder to read because:
 - There's inevitably tons of local variables flying around, making it harder to grok.
 - It's easy to lose track of the point of all the code, as you read the function signature pages and pages ago.
 - It encourages long control loop bodies which become confusing to read (e.g 10s of lines `if` statements make it hard to realise where the `else` block pairs up with, and what the original branch condition was).

Please break it up!

Reply:
It's broken up.